src/aioswitcher/api/__init__.py

```diff
@@ -583,6 +583,7 @@ async def control_breeze_device(
             "logged in session_id=%s, timestamp=%s", login_resp.session_id, timestamp
         )
 
+        cmd_response = None  # type: SwitcherBaseResponse | None
```

Comment:
For future changes it would be better to type it:
```python
cmd_response: SwitcherBaseResponse | None = None
```

Reply:
Actually I already did on my computer earlyer, but for no actual reason, because there's no actual difference between the two.